docs/How-To/cli/README.md

```diff
@@ -187,94 +208,157 @@ fundraisingd tx fundraising cancel 1 \
+| coin         | how many coins to bid. The denom can be both selling and paying coin denom for fixed-price. However, batch-worth must be paying coin denom and batch-many must be selling coin denom |
 
 Example command:
 
 ```bash
+# Place a fixed price bid type for the fixed price auction
 fundraisingd tx fundraising bid 1 fixed-price 1.0 5000000denom2 \
 --chain-id fundraising \
---from steve \
+--from bob \
 --keyring-backend test \
 --broadcast-mode block \
 --yes \
 --output json | jq
+
+# Place a batch-worth bid type for the batch auction
+fundraisingd tx fundraising bid 1 batch-worth 0.35 10000000denom2 \
+--chain-id fundraising \
+--from bob \
+--keyring-backend test \
+--broadcast-mode block \
+--yes \
+--output json | jq
+
+# Place a batch-many bid type for the batch auction
+fundraisingd tx fundraising bid 1 batch-many 0.4 10000000denom1 \
+--chain-id fundraising \
+--from bob \
+--keyring-backend test \
+--broadcast-mode block \
+--yes \
+--output json | jq
+
+#
+# Tips
+#
+# Query all bids that belong to the auction
+fundraisingd q fundraising bids 1 -o json | jq
 ```
 
-### ModifyBid
+## ModifyBid
 
-This is for a bidder to modify an existing bid that the bidder placed previously. Note that this `ModifyBid` is supported only for `BatchAuction`. The bidder can modify the bid only with the same bid type, and also only with either higher bid price or larger bid amount.  
+This command is used for modifying the bid. It is only supported for `BatchAuction`. The bidder is allowed to modify the bid only with the same bid type and they must provide either higher bid price or larger bid amount. Lowering bid price or lesser bid amount is restricted.
 
 Usage
 
 ```bash
 modify-bid [auction-id] [bid-id] [price] [coin]
 ```
 
-| **Argument**      |  **Description**                     |
-| :---------------- | :----------------------------------- |
-| auction-id        | auction id that the bid corresponds to. | 
-| bid-id            | bid id that the bidder placed previously in this auction |
-| price             | bid price (dec type) of a selling coin as the unit of a paying coin.This price must be higher than or equal to the bid price of the previous bid. | 
-| coin              | how many coins to bid, where the denom should be of the paying coin for the bid types of batch-worth, and of the selling coin for the bid type of batch-many. The denom must be the same as the denom of [coin] of the previous bid. |
+| **Argument**|  **Description**              |
+| :---------- | :---------------------------- |
+| auction-id  | auction id                    | 
+| bid-id      | bid id that the bidder placed |
+| price       | bid price of a selling coin as the unit of a paying coin. For `FixedPriceAuction`, it must be the start price of the auction. For `BatchAuction`, the price must be higher than or equal to the minimum bid price of the auction | 
```

Comment:
```suggestion
| price       | bid price of a selling coin as the unit of a paying coin. The price must be higher than or equal to both the original bid price of the auction | 
```

Reply:
Should we not need to explain about price for `FixedPriceAuction`?

Reply:
How about the following change?

> bid price of a selling coin as the unit of a paying coin. For `FixedPriceAuction`, the price must be the same as start price and for `BatchAuction`, the price cannot be lower than the original bid price of the auction.

Reply:
Since `FixedPriceAuction` does not have `ModifyBid`, I don't think we need to explain this for `FixedPriceAuction`. If need, how about mentioning that `FixedPriceAuction` does not support to modify the bid previously placed?

Reply:
Oh I misunderstood this line with `PlaceBid`. I get it now. Let me go make the change.